chart/ory/charts/gcloud-sqlproxy/README.md

```diff
@@ -0,0 +1,145 @@
+$ helm upgrade pg-sqlproxy rimusz/gcloud-sqlproxy --namespace sqlproxy \
+    --set serviceAccountKey="$(cat service-account.json | base64 | tr -d '\n')" \
+    --set cloudsql.instances[0].instance=INSTANCE \
+    --set cloudsql.instances[0].project=PROJECT \
+    --set cloudsql.instances[0].region=REGION \
+    --set cloudsql.instances[0].port=5432 -i
+```
+
+2. Replace Postgres/MySQL host:
+- If access is from the same Namespace, replace with `pg-sqlproxy-gcloud-sqlproxy`.
+- If it is from a different Namespace, replace with `pg-sqlproxy-gcloud-sqlproxy.sqlproxy`.
+
+> **Tip**: List all releases using `helm list`
+
+> **Tip**: If you encounter a YAML parse error on `gcloud-sqlproxy/templates/secrets.yaml`, you might need to set `-w 0` option to `base64` command.
+
+> **Tip**: If you are using a MySQL instance, you may want to replace `pg-sqlproxy` with `mysql-sqlproxy` and `5432` with `3306`.
+
+> **Tip**: Because of limitations on the length of port names, the `instance` value for each of the instances must be unique for the first 15 characters.
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release-name` deployment:
+
+```console
+$ helm delete my-release-name
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the `gcloud-sqlproxy` chart and their default values.
+
+| Parameter                         | Description                             | Default                                                                                     |
+| --------------------------------- | --------------------------------------  | ---------------------------------------------------------                                   |
+| `image`                           | SQLProxy image                          | `gcr.io/cloudsql-docker/gce-proxy`                                                        |
+| `imageTag`                        | SQLProxy image tag                      | `1.17`                                                                                      |
+| `imagePullPolicy`                 | Image pull policy                       | `IfNotPresent`                                                                              |
+| `replicasCount`                   | Replicas count                          | `1`                                                                                         |
+| `serviceAccountKey`               | Service account key JSON file. Provide it encoded with base64 if no existing Secret is used to create a new Secret. | |
+| `existingSecret`                  | Name of an existing secret to be used for the cloud-sql credentials | `""`                                                            |
+| `existingSecretKey`               | The key to use in the provided existing secret   | `""`                                                                               |
```

Comment:
I find both Secret and secret in the text. If there isn't specific reason for that, please be consistent and use Secret or secret. I would use the word with small letters.